socorro/external/boto/connection_context.py

```diff
@@ -14,20 +14,78 @@
 from configman import Namespace, RequiredConfig, class_converter
 
 from socorrolib.lib.converters import change_default
+from socorrolib.lib.ooid import dateFromOoid
```

Comment:
The fact that it's called "ooid" (and the case style of the function) tells me this one is old. Feel free to evaluate if it's worth writing a new one. By the way, the name of a crash ID has changed over the years from ooid to uuid to crash ID.